src/main/java/wolox/training/models/DTO/BookDTO.java

```diff
@@ -0,0 +1,87 @@
+package wolox.training.models.DTO;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+public class BookDTO {
+    private String isbn;
+
+    private String title;
+
+    private String subtitle;
+
+    private List<AuthorDTO> publishers;
+
+    private String publish_date;
+
+    private int number_of_pages;
```

Comment:
As a rule of thumb we name our attributes with camelCase instead of snake_case. If you named it this way because of the json attribute mapping, you can check out this annotation: `@JsonProperty("...")` 🙂